prime-angular-frontend/src/app/shared/components/dialogs/content/claim-enrollee/claim-enrollee.component.ts

```diff
@@ -24,21 +25,41 @@ export class ClaimEnrolleeAction {
 })
 export class ClaimEnrolleeComponent implements OnInit {
   public adjudicators$: BehaviorSubject<Admin[]>;
+  private possibleAction: ClaimActionEnum;
 
   constructor(
+    @Inject(MAT_DIALOG_DATA) public data: DialogOptions,
     private adjudicationResource: AdjudicationResource,
     private dialogRef: MatDialogRef<ConfirmDialogComponent>,
   ) {
+    this.possibleAction = data.data?.possibleAction;
     this.adjudicators$ = new BehaviorSubject<Admin[]>([]);
   }
 
+  /**
+   * @returns Whether to display as a "claim" form (`true`) or as a "disclaim" form (`false`)
+   */
+  public displayAsClaimPrompt(): boolean {
+    return this.possibleAction == ClaimActionEnum.Claim;
+  }
+
+  public getTitle(): string {
+    return (this.possibleAction == ClaimActionEnum.Claim ? "Claim" : "Disclaim") + " Enrollee";
```

Comment:
You can use this ternary directly in the template `{{ ternary }} Enrollee` you just need to add `public ClaimAction = ClaimAction;` at the top of the component to allow enums in the template.